src/Configuration/Dotenv/Parser.hs

```diff
@@ -14,22 +14,24 @@
 
 {-# LANGUAGE CPP #-}
 
-module Configuration.Dotenv.Parse (configParser) where
```

Comment:
Is this a whim? We should not rename modules of published packages like this. This even doesn't help anything and breaks compatibility, this should be renamed back.